packages/web-app-files/src/components/SideBar/Shares/Collaborators/ShowCollaborator.vue

```diff
@@ -265,7 +265,12 @@ export default {
     },
 
     $_reshareInformation() {
-      if (!this.collaborator.resharers) {
+      if (
+        !this.collaborator.resharers ||
+        !this.collaborator.resharers[0] ||
+        !this.collaborator.resharers[0].displayName ||
+        typeof this.collaborator.resharers[0].displayName !== 'string'
```

Comment:
What's the advantage of only checking the first occurance in the array for a displayName of type string instead of validating the whole array of resharer objects? Genuinely asked

Reply:
Also, perhaps you could reflect this in the tests?

Reply:
you are right, I've changed the checks so that every item in the array is checked and if any does not contain a `displayName` or has an invalid displayName `null` is returned